src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -399,6 +400,26 @@ void handle_actor_worker_connect(LocalSchedulerState *state,
   dispatch_actor_task(state, algorithm_state, actor_id);
 }
 
+/**
+ * Finishes a killed task by inserting dummy objects for each of its returns.
+ */
+void finish_killed_task(LocalSchedulerState *state, TaskSpec *spec) {
```

Comment:
When a task is finished in this manner, we should update its state in the task table. It should probably either be marked as DONE or as a new state like KILLED. cc @stephanie-wang